[CircularProgress] Remove static variant, simplify determinate (#22060)
mui-org/material-ui: docs/src/pages/guides/migration-v4/migration-v4.md

  +<Button />
  ```

### CircularProgress

> - The `static` variant has been merged into the `determinate` variant, with the latter assuming the appearance of the former.

Reviewer: ```suggestion
- The `static` variant has been merged into the `determinate` variant, with the latter assuming the appearance of the former.
  The removed variant was rarely useful. It was an exception of Material Design that they have recently removed too from the specification.
```

>   ```diff
>   -<CircularProgress variant="static" />
>   -<CircularProgress variant="static" classes={{ static: 'classname' }} />
>   -<CircularProgress variant="determinate" classes={{ determinate: 'classname' }} />
>   +<CircularProgress variant="determinate" />
>   +<CircularProgress variant="determinate" classes={{ determinate: 'classname' }} />
>   -<CircularProgress variant="determinate" />
>   ```

Reviewer: ````suggestion
  ```diff
  -<CircularProgress variant="determinate" />
  ```

  ```diff
  -<CircularProgress variant="static" classes={{ static: 'className' }} />
  +<CircularProgress variant="determinate" classes={{ determinate: 'className' }} />
  ```
````

### CircularProgress

- The `static` variant has been merged into the `determinate` variant, with the latter assuming the appearance of the former.

>   The removed variant was rarely useful. It was an exception of Material Design that they have recently removed too from the specification.

Author: ```suggestion
  The removed variant was rarely useful. It was an exception to Material Design, and was removed from the specification.
```
Not that recent, and will become less so.

>   The removed variant was rarely useful. It was an exception of Material Design that they have recently removed too from the specification.

Author: ```suggestion
  The removed variant was rarely useful. It was an exception to Material Design, and was removed from the specification.
```
Not that recent, and will become less so.
